cmd/loop/quote.go

```diff
@@ -98,10 +124,39 @@ func quoteIn(ctx *cli.Context) error {
 			"amount.\n")
 	}
 
+	// If the user specified static address deposits, we quoted for their
+	// total value and need to display that value instead of the manually
+	// selected one.
+	if manualAmt == 0 {
+		quoteReq.Amt = int64(depositAmt)
+	}
 	printQuoteInResp(quoteReq, quoteResp, ctx.Bool("verbose"))
 	return nil
 }
 
+func depositAmount(ctx context.Context, client looprpc.SwapClientClient,
+	depositOutpoints []string) (btcutil.Amount, error) {
+
+	var (
+		depositAmt btcutil.Amount
+	)
```

Comment:
`var depositAmt btcutil.Amount`
also it can be moved before the loop, since it is used there.